tests/acceptance/setup-local.js

```diff
@@ -1,13 +1,37 @@
 import { setDefaultTimeout, After, Before } from 'cucumber'
-import { createSession, closeSession, startWebDriver, stopWebDriver } from 'nightwatch-api'
+import { createSession, closeSession, startWebDriver, stopWebDriver, client } from 'nightwatch-api'
+import join from 'join-path'
```

Comment:
shouldn't we use `join` from `helpers/path`? see #2535

Reply:
I was just lazy to import from `./helpers/path`, as the behavior is same. Fixing it.